pymc/testing.py

```diff
@@ -212,6 +212,7 @@ def RandomPdMatrix(n):
 Rplusbig = Domain([0, 0.5, 0.9, 0.99, 1, 1.5, 2, 20, np.inf])
 Rminusbig = Domain([-np.inf, -2, -1.5, -1, -0.99, -0.9, -0.5, -0.01, 0])
 Unit = Domain([0, 0.001, 0.1, 0.5, 0.75, 0.99, 1])
+Unitbig = Domain([0.1, 0.5, 0.75, 0.99, 1])
```

Comment:
The edges aren't included, do to test `0.1` you need to add something before
```suggestion
Unitbig = Domain([0, 0.1, 0.5, 0.75, 0.99, 1])
```

Reply:
Unitbig isn't used now, so I am discarding this.

It was added to test the self-consistency of ICDF-logcdf for the uniform distribution, but we are not using this as they have scipy ppf functions available.